IBX-10224: Describe LTS Updates as regular features (#361)
ezsystems/user-documentation: docs/content_management/block_reference.md

!!! note "Use AI to improve text"

>     If the [AI Actions](ai_actions.md) Lfeature has configured in your application, the **Improve text** button is visible.

Reviewer: ```suggestion
    If [AI Actions](ai_actions.md) have been configured in your application, the **Improve text** button is visible.
```
